Watchman.Discord/Areas/Statistics/Models/MessageInformation.cs

```diff
@@ -9,6 +9,6 @@ public class MessageInformation : Entity
         public MessageInformationChannel Channel { get; set; }
         public MessageInformationServer Server { get; set; }
         public string Content { get; set; }
-        public DateTime Date { get; set; }
+        public DateTime DateTime { get; set; }
```

Comment:
```suggestion
        public DateTime Date { get; set; }
```
Tu mogłoby być jakieś SentAt albo CreatedAt
Ale Date też jest w pełni zrozumiałe